### ISSUE TITLE
very small files are compressed bigger

### ISSUE THREAD

**Issue by redwerkz** [sondr3/astro-compressor#8]
for example this `robots.txt (23 B)` will become `robots.txt.br (27 B)` and `robots.txt.gz (43 B)`
 
> User-agent: *
> Allow: /

**Comment by sondr3** [comment]
Yeah, that's to be expected from very small files. It might be worth it to not compress `.txt` files instead since I doubt they are used as assets outside of `robots.txt`/`humans.txt` etc.

**Comment by redwerkz** [comment]
> It might be worth it to not compress `.txt` files instead 

maybe opt-in `.txt` files for compression

> since I doubt they are used as assets outside of `robots.txt`/`humans.txt` etc.

`security.txt` is another one.

**Comment by sondr3** [comment]
I added a configuration option to change the default files that are compressed, so you can now configure this to your own liking: https://github.com/sondr3/astro-compressor#configuration.
